src/Accordion.php

```diff
@@ -23,412 +16,487 @@
+     *
+     * @param array|string $value The CSS style for the alert component. If an array, the values will be separated by
+     * a space. If a string, it will be added as is. For example, 'color: red;'. If the value is an array, the values
+     * will be separated by a space. e.g., ['color' => 'red', 'font-weight' => 'bold'] will be rendered as
+     * 'color: red; font-weight: bold;'.
+     * @param bool $overwrite Whether to overwrite existing styles with the same name. If `false`, the new value will be
+     * appended to the existing one.
+     *
+     * @return self A new instance with the specified CSS style value added.
      */
-    public function withoutEncodeLabels(): self
+    public function addCssStyle(array|string $value, bool $overwrite = true): self
     {
         $new = clone $this;
-        $new->encodeLabels = false;
+        Html::addCssStyle($new->attributes, $value, $overwrite);
 
         return $new;
     }
 
     /**
-     * List of groups in the collapse widget. Each array element represents a single group with the following structure:
-     *
-     * - label: string, required, the group header label.
-     * - encode: bool, optional, whether this label should be HTML-encoded. This param will override global
-     *   `$this->encodeLabels` param.
-     * - content: array|string|object, required, the content (HTML) of the group
-     * - options: array, optional, the HTML attributes of the group
-     * - contentOptions: optional, the HTML attributes of the group's content
+     * Adds a new item to the accordion.
      *
-     * You may also specify this property as key-value pairs, where the key refers to the `label` and the value refers
-     * to `content`. If value is a string it is interpreted as label. If it is an array, it is interpreted as explained
-     * above.
-     *
-     * For example:
+     * @param string $header The header of the item.
+     * @param string $body The body of the item.
+     * @param bool|string $id The ID of the item. If `true`, an ID will be generated automatically.
+     * @param bool $encodeHeader Whether to encode the header content. Default is `true`.
+     * @param bool $encodeBody Whether to encode the body content. Default is `true`.
      *
-     * ```php
-     * echo Accordion::widget()
-     *     ->items(
-     *         [
-     *             [
-     *                 'Introduction' => 'This is the first collapsible menu',
-     *                 'Second panel' => [
-     *                     'content' => 'This is the second collapsible menu',
-     *                 ],
-     *             ],
-     *             [
-     *                 'label' => 'Third panel',
-     *                 'content' => 'This is the third collapsible menu',
-     *             ],
-     *         ],
-     *     );
-     * ```
+     * @return self A new instance with the specified item.
      */
-    public function items(array $value): self
-    {
+    public function addItem(
```

Comment:
`items()` uses objects (`AccordionItem`), but `addItem()` uses scalars. It's not consistent.

Reply:
How about accepting AccordionItem as well instead of creating and instance in the method?

Reply:
done.